plugins/org.jboss.tools.openshift.common.ui/src/org/jboss/tools/openshift/internal/common/ui/connection/ConnectionWizardPageModel.java

```diff
@@ -216,6 +226,23 @@ private String getSignupUrl(String host, IConnectionFactory factory) {
 		return factory.getSignupUrl(host);
 	}
 
+	private String getUserdocUrl(String host, IConnectionFactory factory) {
+		if (factory == null
```

Comment:
the factory should provide the userdoc link. That should make the link depend on whether OS2 or OS3 has been selected. If null is returned, then nothing is displayed